### DISCUSSION TITLE
Importing _udunits2 on Google Colab

### DISCUSSION THREAD

**Discussion by mg725** [SciTools/cf-units#187] [Q&A]
Hi, I am currently trying to use Iris within Google Colab. I have managed to install it through Conda, however I am struggling to import it - when I do, the following error message is returned:

ImportError                               Traceback (most recent call last) <ipython-input-24-089b5db944e7> in <module>() ----> 1 import iris
6 frames
/usr/local/lib/python3.6/site-packages/cf_units/__init__.py in <module>()      43 from .util import _OrderedHashable, approx_equal      44  ---> 45 from cf_units import _udunits2 as _ud      46 from cf_units._udunits2 import (UT_ASCII, UT_ISO_8859_1, UT_LATIN1, UT_UTF8,      47                                 UT_NAMES, UT_DEFINITION) ImportError: cannot import name '_udunits2' from 'cf_units' (/usr/local/lib/python3.6/site-packages/cf_units/__init__.py)

The problem seems to be that Google Colab is struggling to import _udunits2 from cf_units. I have checked and both these packages are installed on Google Colab and so I can't understand why they won't import.

Does anyone have any ideas of how to get round this issue? Thanks very much in advance if you can help out at all!

**Comment by trexfeathers** [comment]
See #188
